python/openmldb/test/sql_smoke_test.py

```diff
@@ -14,7 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-from openmldb_sdk import sdk as sdk_module
+import sys
```

Comment:
Will it be reformat? You'd better add fmt:off

Reply:
And `../` is not a good way, `sys.path.append(os.path.dirname(__file__) + "/..")` is better?

Reply:
We don't need add "../" now for relative import. @HuilinWu2 please try to remove these.